In [1]:
""" Percentil """

edades = [15, 31, 35, 12, 16, 17, 50, 46, 15, 35, 35, 34, 14, 67, 80, 6, 8, 61, 27, 31]
print(len(edades))

20


In [4]:
""" Necesito el percentil del 75% """
import numpy as np

per75 = np.percentile(edades,75)
print(per75)

per50 = np.percentile(edades,50)
print(per50)

37.75
31.0


Perc75 -> El resultado fue de 37.75, lo que significa que el 75% de las personas tiene 37.75 años o menos.

Perc50 -> 31 años, el 50% de las personas tiene 31 años o menos.